ProcessMaker/processmaker
# Implement Conditional Character Limitation for User Names in Project Details Page

Implement conditional character limitation for user names that exceed a configured length within the Project Details page as Project Members.

## Solution
- Added a new property `characterLimit="INTEGER"` to the `<avatar-image>` tag to manage the character limit for user names.

## How to Test

1. Ensure there are user names that exceed the configured character limit.
2. Go to the Project Details page.
3. Confirm that user names surpassing the configured length are appropriately truncated within the Project Members section.
4. Verify that the truncation of user names doesn't disrupt the general layout and functionality of the Project Details page.

## Related Tickets & Packages
-[ FOUR-11015](https://processmaker.atlassian.net/browse/FOUR-11015)
- [Package PR](https://github.com/ProcessMaker/package-projects/pull/55)

ci:next
ci:package-projects:observation/FOUR-11015

## Code Review Checklist
